code/_helpers/global_lists.dm

```diff
@@ -89,6 +89,45 @@ var/global/list/cloaking_devices = list()
 //Hearing sensitive listening in closely
 var/global/list/intent_listener = list()
 
+// Food
+var/global/list/proteinbar_flavors = list( // we should probably update
```

Comment:
Additionally, this global list should be made into decls.

Reply:
I would, but I'd need some help 'cause I have no idea how decls work and I don't want to break this PR

Reply:
addressed